src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -16,6 +16,106 @@
 		<var name="ns-bibo" value="http://purl.org/ontology/bibo/"/>
 	</vars>
 	<macros>
+		<macro name="setRdfSubjectDescribedBy">
+			<combine name="~rdf:subject" value="${a}">
+				<data source="$[field]"/>
+				<data source="@id" name="a">
+					<regexp match="(.*)" format="$[ns-lobid-resource]${1}"/>
+				</data>
+			</combine>
+		</macro>
+		<macro name="setProvenance">
+			<combine name="$[field_1]" value="$[ns-lobid-organisation]${a}#!" flushWith="$[field_2]">
+				<choose name="a" flushWith="$[field_2]">
+					<data source="$[field_2]" name="a">
+						<lookup in="sigel2isilMap"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- Off = Offline -->
+						<regexp match="[hH][bB][zZ]/[oO][fF][fF]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<!-- OZK = Online-Zentralkatalog" -->
+						<regexp match="HBZ/OZK" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="[hH][bB][zZ]/[rR][eE][dD]" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibAlt" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBibOff" format="DE-605"/>
+					</data>
+					<data source="$[field_2]" name="a">
+						<regexp match="HBZ/NWBib" format="DE-605"/>
+					</data>
```

Comment:
@Everything starting with "HBZ" can be removed because there is a rule at the end that states "if something starts with HBZ => DE-605". (see line 71).
Ah - wait! In line 71 you declared that the entry not only *begins* with HBZ but that that is the complete string ""^HBZ$" (noting the "$" at the end). Ok so, this makes a difference. But are you sure that anything *not* starting with "HBZ" is something different that "DE-605" ? I mean, this is not declared in this morph anyway so it would be save to reduce all "DE-605" matches to one ""^HBZ".